auth-web/src/components/auth/staff/SetupAccountForm.vue

```diff
@@ -2,20 +2,55 @@
   <div>
     <div>
       <v-form ref="directorSearchForm">
-        <v-alert type="error" v-show="errorMessage">{{errorMessage}}</v-alert>
-
         <v-row>
-            <v-col class="d-flex" cols="12" xl="6">
-                <v-text-field
-                filled
-                label="Account Name"
-                v-model.trim="accountName"
-                :rules="accountNameRules"
-                persistent-hint
-                :disabled="saving"
-                />
+            <v-col class="flex pb-0" cols="12">
+              <h4 class="pb-3">Enter a name for this account</h4>
+              <v-text-field
+              filled
+              label="Account Name"
+              v-model.trim="accountName"
+              :rules="accountNameRules"
+              persistent-hint
+              :disabled="saving"
+              />
+            </v-col>
+            <v-col class="flex pb-0" cols="12">
+              <h4 class="pb-1">Account Owner Contact</h4>
+              <p class="body-2 pb-2">Enter the email address of the user who will be managing this account. An email will be sent to this user to verify and activate this account</p>
```

Comment:
Remove the 'body-2' class here. We have some generic styling for paragraph tags in place. We currently don't have overrides for the 'body-2' class and it references the 'Roboto' font family directly.